HTTPServer.test/ServerTests.cs

```diff
@@ -7,11 +7,34 @@ namespace HTTPServer.test
 {
     public class ServerUnitTests
     {
+        Server server = new Server();
+
         [Fact]
         public void ServerCanStart()
         {
-            var server = new Server();
-            Assert.True(server.Start() && server.running == false);
+            Assert.True(server.Start() && server.Running);
+        }
+
+        [Fact]
+        public void ServerCanStop()
+        {
+            server.Start();
+            Assert.True(server.Stop() && !server.Running);
+        }
+
+        [Fact]
+        public void ServerCanRespond200()
+        {
+            FileConnection file = new FileConnection();
```

Comment:
Since I am using input and output from files to imitate sockets it seems necessary.  If I can't do it in the unit tests, then that seems to leave the source code, which is bad because it will create unnecessary dependencies.  Are there other options I am missing?